dcmdata/libsrc/dclist.cc

```diff
@@ -155,6 +171,7 @@ DcmObject *DcmList::insert( DcmObject *obj, E_ListPos pos )
                 node->nextNode = currentNode;
                 currentNode->prevNode = node;
                 currentNode = node;
+                currentAbsolutePosition++;
```

Comment:
Isn't this increment wrong as the new node gets the position of the current node?

Reply:
Good catch! Not sure what I was thinking. 😬 

Added test criteria, and fixed the problem.

Reply:
Thank you for confirming my observation and also for the updated regression test.